### PR TITLE
Dildos

### PR DESCRIPTION
**Pull request by AliiTea**
:cl: Nexendia
add: I added the fucking dildos for you fucks, praise me.
/:cl:

### CODE REVIEW

**Comment by TDcoolguy300** [comment]
Negative

**Comment by AliiTea** [comment]
nofun

**Review by Carbonhell — APPROVED** [review]

**Comment by KayeArray** [comment]
it's fucking happening

File: hippiestation/code/game/objects/items/dildo.dm [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+/obj/structure/statue/dragon/shelf
+	name = "Dragon Dildo Shelf"
+	desc =  "Built to withstand your collection and your sins."
+	icon = 'hippiestation/icons/obj/dicks.dmi'
+	icon_state = "shelf1"
+
+/obj/structure/statue/dragon/shelf/alt
+	name = "Dragon Dildo Shelf"
+	desc =  "Built to withstand your collection and your sins."
```
Comment (GuyonBroadway) [inline hippiestation/code/game/objects/items/dildo.dm]: if this is a child of shelf there is no need to re-declare the name and description.

**Comment by GuyonBroadway** [comment]
Also, put yer name in the changelog else nobody can priase you.

Also up the embed chance because why not?

**Comment by TDcoolguy300** [comment]
Negative

**Pull request merged**